app/helpers/images_helper.rb

```diff
@@ -3,7 +3,7 @@ def image_absolute_url(image, version)
     return "" unless image
 
     if Paperclip::Attachment.default_options[:storage] == :filesystem
-      URI(request.url) + image.attachment.url(version)
+      URI(request.url) + image.url(version)
     else
       investment.image_url(version)
```

Comment:
This line is broken as there is no investment defined within this context. This throws always an exception when using a paperclip remote storage.

```suggestion
      image.url(version)
```